### DISCUSSION TITLE
Compositing

### DISCUSSION THREAD

**Discussion by Cjjuntti** [keiwando/toonsquid#577] [Ideas]
So, hear me out a moment. Toonsquid is an amazing app, but I feel like its potential hasn’t been fully realized yet. I think something that would be cool to have in the app is the ability to composite and have fx layers that you can parent to other layers via the transform hierarchy, kind of like fx nodes in toon boom or opentoonz. I know this probably seems like a long way away, but I would love to see something like this implemented over time in the app, even if it takes months, or even years. If anyone could give me any thoughts about this, I’d love to hear it. Let me know what you think.

**Comment by DinoAwakening** [comment]
2.0 update is on the way. We don’t know if that feature you’re talking about was already implemented and waiting to be released. Sounds like a cool idea, and I hope it comes in 2.0 :)

**Reply by Cjjuntti** [reply]
Dude, if compositing is gonna be in the next update, that would be awesome. Even if it doesn’t, toonsquid will still be amazing regardless. If it doesn’t come in 2.0, we can always hope for it to be in another update.
